scripts/mkcloud

```diff
@@ -72,6 +72,7 @@ nodenumbercomputedefault=2
 # expect to have this many physical machines attached via $mkch_physcloudif
 # these replace virtual machines, so we start less VMs
 : ${nodenumberphys:=0}
+. $qa_crowbarsetup
```

Comment:
i think this is a bit hidden. maybe we can add a source function which will have all sourcing code. as we are sourcing the libs in another place. i would put that together.